src/App.vue

```diff
@@ -262,6 +262,7 @@ export default class App extends Mixins(BcolMixin, DateMixin, FilingTemplateMixi
       // process errors/warnings
       switch (error?.response?.status) {
         case PAYMENT_REQUIRED:
+          this.haveChanges = false
```

Comment:
Please add a comment why it's OK to clear this flag in this situation.

Reply:
(The principle I'm following is that code paths are often symmetric/similar, and in this case they're not. As a reviewer, I wonder why they're not. Totally fair scenario, just document it.)

Reply:
Perfect, thanks!